docs/tutorial/offscreen-rendering.md

```diff
@@ -1,59 +1,66 @@
 # Offscreen Rendering
 
-Offscreen rendering lets you obtain the content of a browser window in a bitmap,
-so it can be rendered anywhere, for example on a texture in a 3D scene. The
-offscreen rendering in Electron uses a similar approach than the [Chromium
+## Overview
+
+Offscreen rendering lets you obtain the content of a browser window in a
+bitmap, so it can be rendered anywhere, for example, on texture in a 3D scene.
+The offscreen rendering in Electron uses a similar approach than the [Chromium
 Embedded Framework](https://bitbucket.org/chromiumembedded/cef) project.
```

Comment:
```suggestion
Offscreen rendering lets you obtain the content of a `BrowserWindow` in a
bitmap, so it can be rendered anywhere, for example, on texture in a 3D scene.
The offscreen rendering in Electron uses a similar approach to that of the [Chromium
Embedded Framework](https://bitbucket.org/chromiumembedded/cef) project.
```